clog/global_state.py

```diff
@@ -55,34 +55,25 @@ def check_create_default_loggers():
                 raise ValueError('log_dir not set; set it or disable clog_enable_file_logging')
             loggers.append(FileLogger())
 
-        stream_backend_map = create_stream_backend_map()
-
-        # possibly add logger that writes to scribe
         if not config.scribe_disable:
-            logger = ScribeLogger(
+            scribe_logger = ScribeLogger(
                 config.scribe_host,
                 config.scribe_port,
-                config.scribe_retry_interval,
-                stream_backend_map=stream_backend_map
+                config.scribe_retry_interval
             )
-            loggers.append(logger)
-
-        if config.clog_enable_stdout_logging:
-            loggers.append(StdoutLogger())
-
-        if not config.monk_disable:
-            # TODO: this check should be removed once most libraries have
-            # been moved to use the "internal" extra-requires.
-            if monk_dependency_installed:
-                logger = MonkLogger(
-                    config.monk_client_id,
-                    stream_backend_map=stream_backend_map
+            if not config.monk_disable and monk_dependency_installed:
+                scribe_monk_logger = ScribeMonkLogger(
```

Comment:
Is it expected that we need to have `scribe_disable: False` to be able to log using Monk?

I'm fine with that, but it might make it trickier to fully transition to Monk in the future